add footer.html.erb

ChenKLi opened the pull request:
1. Create add footer.html.erb file
2. Layout footer without using any CSS framework.
🤚 We have to decide 1 or 2 CSS framework, which we prefer to use in this project.

ChenKLi committed:
add footer.html.erb

ChenKLi committed:
Update footer.html.erb

fix css selector (line number 47) 
footer .footer_link -> .footer .footer_link

cin-yi-wei reviewed (commented):
footer_logo 感覺要放在 footer_list裡面
變成四欄的表格。如果要改寬度可以用flex-grow / flex-shrink來調整。

ChenKLi committed:
Update footer.html.erb

HTML 
1. move .footer_logo into .footer_list 
CSS (style)
1. a tag hover color replace font-weight  (line number 65)

ChenKLi commented:
> footer_logo 感覺要放在 footer_list裡面 變成四欄的表格。如果要改寬度可以用flex-grow / flex-shrink來調整。

 👍 已修正

cin-yi-wei reviewed (approved):
check ok

zoechaoweiting reviewed (commented):
footer_link裡面的那些a標籤如果外面再用ul>li包起來會更好～
語意上會比單純只有div清楚

ChenKLi commented:
> footer_link裡面的那些a標籤如果外面再用ul>li包起來會更好～ 語意上會比單純只有div清楚

OK，這個部份我修正一下
